util/fibers/uring_file.h

```diff
@@ -75,6 +79,11 @@ class LinuxFile {
   void WriteFixedAsync(io::Bytes src, off_t offset, unsigned buf_index, AsyncCb cb);
   void WriteAsync(io::Bytes src, off_t offset, AsyncCb cb);
   void FallocateAsync(int mode, off_t offset, off_t len, AsyncCb cb);
+
+#ifndef __APPLE__
+  std::error_code StatX(std::string_view filepath, struct statx *stat);
```

Comment:
This interface is kinda bad, because it accepts the `filepath` which should be stored in `UringFile` but we don't 🤷‍♂️

Reply:
and btw, we do not have to implement all the functions possible, imho. The reason we use io_uring is for:
1. asynchronisity
2. low latency.

So if statx is being called fairly rarely - you could just use libc variant if you prefer.